tests/unit/amo/pages/TestCategory.js

```diff
@@ -749,4 +795,52 @@ describe(__filename, () => {
       expect(root.find(NotFound)).toHaveLength(0);
     });
   });
+
+  it('renders a 404 when clientApp is Android and enableFeatureStaticThemesForAndroid is false', () => {
```

Comment:
same here - should you mention that this is only for a theme in the test description?